### ISSUE TITLE
Asset Sharing Across Sites/Spaces

### ISSUE THREAD

**Issue by dmolsen** [wvuweb/cleanslate-cms#126]
UX and development work to support asset sharing across sites will occur during Q2 this year. I believe the simple notion of "sharing assets" will spin off several related features and will take up the vast bulk of our time. Prior to Q2 the internal dev team will lay out an initial feature set and workflow as well as identify problem areas. Once this is done I think we should go to wireframe mock-ups that we can share with prof techs to at least get feedback on flow. I don't want to commit much code that will end up being scrapped.

If there are wish list features for this topic feel free to add comments to this issue. Please note that they won't be addressed until April 1 as we start our Q2 development window.

**Comment by jasonmarkle** [comment]
This is pretty essential to what I'm doing with the plans for Statler's overhaul. We have one blog that needs to be fed to about 6 different sub domains, and would also probably be helpful to be fed to additional sources offsite such as WVU Today.

I guess my wish list item is to have a fully functional blog that can be pulled cross site by category, since not every site would need to pull every category. I know that this can be achieved with tags, but only on a single site. 

The real benefit of this feature is that it exponentially increases workflow. Stories can be posted one time and go to 4 or 5 or 10 places instead of being copied and pasted X times as static html in each page's content area. So let's say each time you need to copy a story statically it takes 3 minutes and you have to do it 20 times a day, there goes an hour you could have spent doing something more productive. 5 hours a week, roughly 260 hours a year spent copying and pasting static HTML content (give or take).

**Comment by dmolsen** [comment]
@jmarkle825 -

FWIW, we're starting the discussion on this today. I've been out so I've held up the start of them.

**Comment by jasonmarkle** [comment]
Yeah, that's why I posted. Adam just told me about it so hopefully that will help some. Our current site is massive, and not all of the content will be there on the new site hopefully. Regardless, the easier it is to manage our audience' main attraction, the more time I can devote to making sure the site rocks.
